### PR TITLE
Update error data path setting

### PR DESCRIPTION
**Pull request by AhmetTanakol**
For nested objects validation, data error path is reconfigured

### CODE REVIEW

**Comment by edgarmueller** [comment]
@AhmetTanakol Thanks for the PR! Would you mind also adding a test case? You may just use the person example with the `personalData` set to empty an object like in the example and then check whether the validation errors pop up.

**Comment by AhmetTanakol** [comment]
test case added

File: example/templates/person.ts [hunk 1]
```diff
@@ -118,7 +118,8 @@ const uischema = {
 const data = {
   name: 'John Doe',
   vegetarian: false,
-  birthDate: '1985-06-02'
+  birthDate: '1985-06-02',
+  personalData: {}
```
Comment (eneufeld) [inline example/templates/person.ts]: @edgarmueller shouldn't nested objects be created on the fly when needed?

Comment (edgarmueller) [reply example/templates/person.ts]: @eneufeld Depends on how you define 'needed': The object is already created once you touch any of its properties, but one might also argue that we should already create an empty object even in case it's not actually required: if that's what you want, I'd propose to move that discussion to a different issue.

**Comment by edgarmueller** [comment]
Looks good. If @eneufeld has no objects we can merge this.

File: example/templates/person.ts (hunk 1 above)
Comment (eneufeld) [reply example/templates/person.ts]: I opened an issue to discuss this #708
